backend/README.md

```diff
@@ -0,0 +1,29 @@
+# MolEvolvR Backend
+
+The backend is implemented as a RESTful API over the following entities:
+
+- `User`: Represents a user of the system. At the moment logins aren't
+required, so all regular users are the special "Anonymous" user. Admins
+have individual accounts.
+- `Analysis`: Represents an analysis submitted by a user. Each analysis has a unique ID
+and is associated with a user. analyses contain the following sub-entities:
```

Comment:
Very persnickety comment, but perhaps all of these sub-entities should be have the `Analysis`- prefix. Or in cases where "Analysis" is already "in the namespace", you could omit it, e.g. in types:

```ts
type Analysis = {
  submission: Submission; // i guess what i'm calling "inputs" in the frontend so far
  status: Status;
  result: Result;
}
```

I'm going to try to keep type, variable, and in-UI names consistent with what you use on the backend, so this will trickle down.

Reply:
Erm, sorry about that README; I've changed nearly all the content there in the upcoming PR, and the part you've commented on isn't there anymore. Perhaps let's just design the thing first and then write the documentation for it once things are more stable.